### ISSUE TITLE
Headings wrongly used for styling

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#1598]
Headings - we are using them incorrectly. No H1, some random H3 and then H6 on the same page. For people navigating with headings it’s a difficult situation. We are using them for styling and there should be used to give corect DOM structure. 

Priority: low impact but quite basic mistake so we should fix this.
